doc/ASCII_ART.md

```diff
@@ -1,4 +1,5 @@
 ﻿## Making ASCII art for CDDA
+ASCII art only requires you to have the json files for the art you want in the `data/json/ascii_art/` folder.  The `"id"` string must match the `"id"` of the item or monster you want to have the art displayed for.
 ### ASCII art Editors
 ASCII art can be made using any text editor, such as Notepad, but it's more convenient to use an editor created for making ASCII art. There are many ASCII art editors on the internet, but this guide focuses on REXPaint because aside from many useful features like drawing ellipses, rectangles, lines and other shapes, it's the only editor with the option to export images to a format readable by CDDA. Note that this is not a comprehensive REXPaint manual, you can find the full manual [here](https://github.com/Lucide/REXPaint-manual/blob/master/manual.md) or in your REXPaint folder.
```

Comment:
```suggestion
ASCII art can be made using any text editor, such as Notepad, but it's more convenient to use an editor created for making ASCII art. There are many ASCII art editors on the internet, but this guide focuses on REXPaint because aside from many useful features like drawing ellipses, rectangles, lines and other shapes, it's the only editor with the option to export images to a format readable by CDDA. Note that this is not a comprehensive REXPaint manual, you can find the full manual [here](https://github.com/Lucide/REXPaint-manual/blob/master/manual.md) or in your REXPaint folder. (Note: REXPaint is only available for Windows, although it should run using Wine on Linux or MacOS.)
```

Reply:
I plan to update this documentation with much more information  - including a tutorial on Playscii which is crossplatform and has much more powerful image creation capabilities.  REXPaint sucks imo

Reply:
BTW, a way to generate at least a first draft of an ascii art set that I was considering: Take a tileset, palletize to the available ascii art palette, use an appropriate [pixel scaling algorithm](https://en.wikipedia.org/wiki/Pixel-art_scaling_algorithms), then converting using something like cacaview or AAlib.

Reply:
Playscii has a really good image to ascii function - it's how I made all my ascii art like those monsters ascii arts.  It only took me a minute to make them from black and white pictures.  Although you do need to have a ascii set character that matches the one in the game, so it knows what characters to convert it to.

Although if tile sets are only 32x32 it would fit the size of the ascii art window very easily and yeah, there could be a pixel to ascii colour block image converter.  The only drawback is that pixels are square, but ASCII characters are rectangular - 1x2 so everything would be squashed out of proportion.

Reply:
After doing some googling on cacaview/aalib/libcaca ...  I think yeah, perhaps I'll give that a shot at some point